build/global.json

```diff
@@ -1,5 +1,6 @@
 {
   "sdk": {
-    "version": "2.1.802"
+    "version": "2.1.803",
```

Comment:
Beleive we need 3.x SDK (think it came in 3.0.100) so we should probably update to latest LTS SDK which is 3.1.301.